README.markdown

```diff
@@ -2,16 +2,17 @@
 [![codecov](https://codecov.io/gh/zfsonlinux/zfs/branch/master/graph/badge.svg)](https://codecov.io/gh/zfsonlinux/zfs)
```

Comment:
Can we rename this file to README.md as we are following in all other places?

Reply:
we can rename it, but i don't think it is required. Extension of the file can be different. If you want we can rename it in a different PR.